src/http_server/cargo_dash_http_server.py

```diff
@@ -0,0 +1,124 @@
+import asyncio
+from aiohttp import web
+
+routes = web.RouteTableDef()
+
+
+@routes.get('/allnodes')
+async def get_all_nodes(request):
+    ''' Receives a get request that asks for all nodes.
+
+        Returns array with all nodes in json format.
+
+        Ex. returns
```

Comment:
"For example" kort je af met e.g. of i.e.

```suggestion
        Eg. returns
```